workspaces/docs/docusaurus.config.js

```diff
@@ -1,57 +1,57 @@
 // @ts-check
 // Note: type annotations allow type checking and IDEs autocompletion
 
-const lightCodeTheme = require('prism-react-renderer/themes/github');
-const darkCodeTheme = require('prism-react-renderer/themes/dracula');
+const lightCodeTheme = require("prism-react-renderer/themes/github");
```

Comment:
We're introducing a-lot of unnecessary changes in this file, I believe changes in this file should only be scoped to #149

Reply:
I think the code formatting tool I use is reducing indentation spaces, which is watched by git. I'll take care of this.

Reply:
Shall I undo it and push a commit?

Reply:
That's okay, we can leave it as it. I'll need to get some ESLint rules in this directory to help with code styles but that will come at another time. 🙂